lib/src/rules/always_declare_return_types.dart

```diff
@@ -64,17 +64,13 @@ class AlwaysDeclareReturnTypes extends LintRule {
 }
 
 class _Visitor extends SimpleAstVisitor<void> {
-  static const LintCode functionCode = LintCode(
-      "always_declare_return_types", // ignore: prefer_single_quotes
+  static const LintCode functionCode = LintCode('always_declare_return_types',
       "The function {0} should have a return type but doesn't.",
-      correction:
-          "Try adding a return type to the function."); // ignore: prefer_single_quotes
+      correction: 'Try adding a return type to the function.');
```

Comment:
This was mimicking the convention in the `analyzer` package that error codes always use double quotes to that we don't have to switch back and forth as we add or remove single quotes within the messages. I think we want to continue using that convention because it makes code reviews easier. (Changing the quotes on the lint name is fine though, because we don't use single quotes in lint names so it doesn't have the same issue.)

Reply:
I see, thanks for your feedback.